### ISSUE TITLE
Documentation for generateFetchers configuration

### ISSUE THREAD

**Issue by olof-nord** [fabien0102/openapi-codegen#39]
Hello @fabien0102, really liking this library!

I noticed that there are three typescript code generation functions possible: `generateSchemaTypes`, `generateReactQueryComponents` and `generateFetchers`.

The two first ones have nice documentation in the cli readme.

How would an example `generateFetchers` configuration look like?

If you could add one here, I could add it to said readme with a PR later.

**Comment by fabien0102** [comment]
Hi 👋 

`generateFetchers` have the same API as `generateReactQueryComponents`, here a real life example:

```ts
// openapi-codegen.config.ts
import { defineConfig } from "@openapi-codegen/cli";
import {
  generateFetchers,
  generateSchemaTypes,
} from "@openapi-codegen/typescript";

export default defineConfig({
  auth: {
    from: {
      source: "file",
      relativePath: "../openapi/bundled/private_auth.yaml",
    },
    outputDir: "auth",
    to: async (context) => {
      const filenamePrefix = "auth";
      const { schemasFiles } = await generateSchemaTypes(context, {
        filenamePrefix,
      });
      await generateFetchers(context, {
        filenamePrefix,
        schemasFiles,
      });
    },
  },
});
```

Keep in mind that the config is in typescript, so you should have autocompletion and documentation on each props:
![image](https://user-images.githubusercontent.com/1761469/155707073-ef7abead-91d5-4f47-bf67-36432d0f6e7c.png)

**Comment by fabien0102** [comment]
This will be solved by #1

**Comment by olof-nord** [comment]
Since youre asking, I actually got one small comment: The generated Fetchers are depending on the `qs` library: I think it would make sense to add a small note about this in the readme, as to know what to install to get started. The dependency on react-query is more or less given, but qs perhaps less so.

Getting started: 
1) install `npm install qs react-query`
2) Then install `npm install --save-dev @openapi-codegen/cli @openapi-codegen/typescript`
